src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let off = 0;
+
+  if (days >= 3 && days < 7) {
+    off = 20;
+  }
+
+  if (days >= 7) {
+    off = 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean